zebra-consensus/src/block.rs

```diff
@@ -8,9 +8,12 @@
 //! Verification is provided via a `tower::Service`, to support backpressure and batch
 //! verification.
 
+mod checks;
+
 #[cfg(test)]
 mod tests;
 
+use checks::*;
```

Comment:
```suggestion
```

It's good to avoid glob imports like this unless they're really necessary, because it makes it harder to keep track of what items are in the local namespace.  Instead of doing a glob import, if the `checks` module is renamed to `check`, then the code below could be

```
check::is_coinbase_first(&block);
```
which is nice because the two parts of the name fit together.  Then when there are more checks, we can read off the code as a sequence of instructions:
```
check::is_coinbase_first(&block)?;
check::something_else(...)?;
check::another_property(...)?;
```

Reply:
Nice one, applied at https://github.com/ZcashFoundation/zebra/pull/917/commits/9df6cf2b6da36979634cbc0a1d2e74f5d0272a8b